res/xml/nepali_phonetic_npl.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<keyboard name="Nepali phonetic" script="nepali">
+  <row>
+    <key key0="ए" key2="ऐ" key4="esc" key7="१"/>
+    <key key0="उ" key2="ऊ" key3="\@" key7="२"/>
```

Comment:
Layout includes some ASCII punctuation but not all, missing: `.`, `<`, `>`, `?`, `` ` ``, `|`, `~`.
Are these useful in your language ?

Reply:
i added some of the punctuation you suggested and full stop "." Is not used in nepali.

Reply:
Add only what is useful!